quiet.mk

```diff
@@ -24,12 +24,12 @@ ifneq ($(VERBOSE),1)
   # $(2): Logfile name
   # $(3): Textual description of the task
   define build
-	@echo -n "Building $(strip $(3)) [$(HL)$(CURDIR)/$(strip $(2))$(NC)] ..."
+	@echo $(if $(filter -j%,$(MAKEFLAGS)),,-n) "Building $(strip $(3)) [$(HL)$(CURDIR)/$(strip $(2))$(NC)] ..."
 	$(strip $(1)) >> $(strip $(2)) 2>&1; \
 	(ERR=$$?; if [ $$ERR = 0 ]; then \
-		echo " $(GREEN)OK$(NC)"; \
+		echo "$(if $(filter -j%,$(MAKEFLAGS)),Building $(strip $(3)) [$(HL)$(CURDIR)/$(strip $(2))$(NC)] ...,) $(GREEN)OK$(NC)"; \
```

Comment:
I would change to

```
Building <lib>library [<log>] ....
Build <lib>library [<log>] OK
Build <lib>library [<log>] FAILED
```

It feels weird to print "Building ...." twice, this way it is clear that one message is when it started and the other is when it finished.

Note to myself: update CI to filter new messages.

Reply:
@SRaus would this change break any CI? If so, can we update it easily?

Reply:
It should not break anything  on CI side.